.github/workflows/eslint.yml

```
@@ -0,0 +1,27 @@
+name: eslint
+on:
+  pull_request:
+    paths-ignore:
+      - '.github/**'
+      - '!.github/workflows/eslint.yml'
+      - '**.md'
+    types: [opened, synchronize]
+
+jobs:
+  eslint:
+    name: eslint
+    runs-on: ubuntu-latest
+    if: github.event.pusher.name != 'dreamkast-cloudnativedays'
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Node.js 16.18.0
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.18.0
```

```suggestion
          node-version: '18.x'
```

cdkに合わせたほうがよさそうですが16にしたい理由があれば知りたい

---

こっちのversionに合わせて設定してたんですが、cdkは18になってたんですね...

https://github.com/cloudnativedaysjp/dreamkast-functions/blob/chore/add_eslint_to_ci/package-lock.json#L29